deep_agent/src/personalization/repository.py

```
@@ -122,7 +122,15 @@ async def delete_memory(self, user_id: str, memory_id: uuid.UUID) -> bool:
                 (str(memory_id), user_id),
             )
             await conn.commit()
-            return bool(cur.rowcount > 0)
+            deleted = bool(cur.rowcount > 0)
+        if deleted:
+            try:
+                from deep_agent.src.cache.personalization_cache import invalidate
+
+                await invalidate(user_id)
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)